src/order/bounded_lattice.lean

```diff
@@ -290,6 +290,10 @@ instance bounded_distrib_lattice_Prop : bounded_distrib_lattice Prop :=
   bot          := false,
   bot_le       := @false.elim }
 
+instance linear_order_Prop : linear_order Prop :=
```

Comment:
In which namespace are we? I would expect this instance to have name `_root_.Prop.linear_order`.

Reply:
We're in root. I'll rename the definition.